src/main/java/uk/gov/companieshouse/psc/extensions/api/mongo/document/InternalData.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.companieshouse.psc.extensions.api.mongo.document;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class InternalData {
```

Comment:
This class also must override equals() and hashCode() since the `PscExtension` class is dependent on them in its own equals() and hashCode().
Apologies if I wasn't clear in the previous comment.